src/core/AutomationClip.cpp

```diff
@@ -1004,17 +1000,14 @@ void AutomationClip::resolveAllIDs()
 	TrackContainer::TrackList l = Engine::getSong()->tracks() +
 				Engine::patternStore()->tracks();
 	l += Engine::getSong()->globalAutomationTrack();
-	for( TrackContainer::TrackList::iterator it = l.begin();
-							it != l.end(); ++it )
+	for (auto& track : l)
 	{
-		if( ( *it )->type() == Track::AutomationTrack ||
-			 ( *it )->type() == Track::HiddenAutomationTrack )
+		if (track->type() == Track::AutomationTrack || track->type() == Track::HiddenAutomationTrack)
 		{
-			Track::clipVector v = ( *it )->getClips();
-			for( Track::clipVector::iterator j = v.begin();
-							j != v.end(); ++j )
+			Track::clipVector v = track->getClips();
+			for (auto& clip : v)
 			{
-				AutomationClip * a = dynamic_cast<AutomationClip *>( *j );
+				AutomationClip* a = dynamic_cast<AutomationClip*>(clip);
 				if( a )
 				{
 					for( QVector<jo_id_t>::Iterator k = a->m_idsToResolve.begin();
```

Comment:
What happened most likely was that I forgot to add ``--format-style=file`` the first time around, so when it ran it didn't consider formatting the changes :sweat_smile:. I could've just reran it with the format style set, but I figured I would do it manually because there were more issues you pointed out that needed to be addressed than just the formatting that clang-tidy wouldn't resolve (i.e, m_ prefix removal, clang-tidy not making the optimal variable name change within the range loop, etc).

Reply:
Sounds reasonable to me. I only spotted this one because the top of the loop showed up in the diff; I bet there are a bunch more out there that it hasn't touched. It's probably not worth scouring the code for every last one, so I won't worry about it further.